src/zeek-setup.cc

```diff
@@ -183,6 +184,7 @@ zeek::plugin::Manager* zeek::plugin_mgr = nullptr;
 zeek::detail::RuleMatcher* zeek::detail::rule_matcher = nullptr;
 zeek::detail::DNS_Mgr* zeek::detail::dns_mgr = nullptr;
 zeek::detail::TimerMgr* zeek::detail::timer_mgr = nullptr;
+zeek::detail::ActivationManager* zeek::detail::activation_mgr;
```

Comment:
`= nullptr`

Reply:
Rut-row!  A holdover from when it was a `unique_ptr` not requiring initialization.  Thanks for catching!